cmd/clusterctl/internal/test/fake_objects.go

```diff
@@ -59,6 +60,16 @@ func NewFakeCluster(namespace, name string) *FakeCluster {
 	}
 }
 
+// NewFakeClusterWithForceLabel is same to NewFakeCluster except it will also return a secret
+// named <clustername>-cloud-config with clusterctl.cluster.x-k8s.io/move label
+func NewFakeClusterWithForceLabel(namespace, name string) *FakeCluster {
```

Comment:
What about implementing 
```
func (f *FakeCluster) WithCloudConfigSecret(fakeControlPlane *FakeControlPlane) *FakeCluster {
	f. withCloudConfigSecret = true
}
```
Instead of using a dedicate constructor? so we are more consistent with all the other methods adding objects

Reply:
yes, will do